internal/summary/summary.go

```diff
@@ -24,29 +33,87 @@ func NewSummary() *Summary {
 }
 
 // Gather collates data into Summary
-func (s *Summary) Gather(services []model.Service) {
+func (s *Summary) Gather(services []model.Service) error {
 
 	for _, svc := range services {
-		var path string
 		// Assume that, in practice, there is only one mount
+		path := ""
 		if len(svc.Mounts) > 0 {
 			path = svc.Mounts[0].Repo.LocalPath
-		} else {
-			path = ""
 		}
-		s.Services = append(s.Services, &Service{
-			Name: string(svc.Name),
-			Path: path,
-		})
+
+		svcSummary := &Service{
+			Name:    string(svc.Name),
+			Path:    path,
+			k8sData: []k8sDatum{},
+		}
+
+		kubeDatum := &k8sDatum{}
+
+		entities, err := k8s.ParseYAMLFromString(svc.K8sYaml)
+		if err != nil {
+			return err
+		}
+
+		lbs := k8s.ToLoadBalancers(entities)
+
+		for _, lb := range lbs {
```

Comment:
any reason not to just do `kubeDatum.lbs = k8s.ToLoadBalancers(entities)`?

or better yet, save yourself a line of code with:
```
kubeDatum := &k8sDatum{
    lbs: k8s.ToLoadBalancers(entities)
}
```